src/components/Categories.js

```diff
@@ -1,7 +1,19 @@
+import { useDispatch } from 'react-redux';
+import { getStatus } from '../redux/categories/categories';
+
 function Categories() {
+  const dispatch = useDispatch();
+  const clicklHandler = () => {
+    dispatch(getStatus());
+  };
   return (
     <section className="categories--container">
-      <button type="button">Check satus</button>
+      <button
+        type="button"
+        onClick={clicklHandler}
+      >
+        Check satus
+      </button>
```

Comment:
Please ensure that the checkCategoriesStatus  dispatches the `underConstruction` at the moment nothing shows on the screen when I click the `check status button`. Kindly fix that.